CHANGELOG.md

```diff
@@ -50,15 +50,27 @@ Initial release of phac-nml/mikrokondo. Mikrokondo currently supports: read trim
 
 - Added integration testing using [nf-test](https://www.nf-test.com/).
 
+
+### `Unreleased`
```

Comment:
Could you make this section at the same level as the other releases and move all the bullet points underneath **Unreleased**?